README.md

```diff
@@ -28,31 +28,35 @@ One or more of the following processors:
 
 - 2080 MiB
 
+### Runtime linux requirements
+
+On Linux platforms with hybrid Nvidia GPU setup please use Nvidia driver R525 or newer. Older ones are known to have compatibility issues. Non hybrid cards are confirmed to be working with R520 and older versions.
+
 ---
 
 ## Build System Requirements
 
 ### All Platforms
 
-- For building CUDA support: NVIDIA Cuda Toolkit 11, an NVIDIA GPU with CUDA support, and an Nvdia driver version R450 or newer.
+- For building CUDA support: NVIDIA Cuda Toolkit 11, an NVIDIA GPU with CUDA support, and an Nvdia driver version R525 or newer.
 - For building Vulkan support: Vulkan SDK 1.3 and a GPU with Vulkan 1.3 runtime support.
 
 ### Windows
 
-- Windows 10 Pro.
+- Windows 10/11 Pro.
```

Comment:
Why Pro?

Reply:
That's a good question, kinda copied that from original 10Pro. Will adjust to "any" windows :)